examples/gno.land/p/demo/gnofundme/gnofundme.gno

```diff
@@ -0,0 +1,124 @@
+package gnofundme
+
+import (
+	"std"
+	"strconv"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/avl"
+)
+
+type Campaign struct {
+	title        string
+	description  string
+	goal         uint
+	current      uint
+	begin        time.Time
+	deadline     time.Time
+	owner        *User
+	beneficiary  *User
+	contributors []*Contributor
+}
+
+func NewCampaign(title string, description string, goal uint, begin time.Time, deadline time.Time, owner *User, beneficiary *User) *Campaign {
+	return &Campaign{
+		title:        title,
+		description:  description,
+		goal:         goal,
+		current:      0,
+		begin:        begin,
+		deadline:     deadline,
+		owner:        owner,
+		beneficiary:  beneficiary,
+		contributors: make([]*Contributor, 0),
+	}
+}
+
+func (c Campaign) GetTitle() string {
```

Comment:
Recommendation: To help yourself (and others) to read your code, I would recommend adding the get functions in the end, or at least make them a whole block (this would mean moving the `GetContributors` function to this block of gets.

Reply:
thanks, didn't pay attention to that, as I added that function later.